app/controllers/search_organisation.js

```diff
@@ -3,39 +3,39 @@ import searchModule from "./search_module";
 
 export default searchModule.extend({
```

Comment:
since you're using the new list, you can remove the searchModule inheritance.
It's also worth considering whether we want to make this generic in some way (there's quite a few that are exactly like this one)

Reply:
Yes, you are right. I thought to add some common code later in `searchModule` and refactor it later when all pages using searchModule have been modified to new list.

Reply:
It's probably better to let searchModule die, and re-create a new generic controller or component from scratch 😄 
that'll save you a lot of trouble

Reply:
Agree with @patrixr